src/js/ctrl/export.ts

```diff
@@ -47,19 +51,20 @@ export function exporter(internal: boolean = false): Spec {
   spec.data = exporter.pipelines(state, int);
 
   // Add interactions and widgets from store
-  spec = exporter.interactions(state, spec);
+  spec = history ? spec : exporter.interactions(state, spec);
```

Comment:
why make an exception for interactions? when someone reverts to a prior history before they added interactivity, won't they want that interactivity to be gone?

Reply:
so this was causing the thumbnails to be interactive too. So when you try to drag and drop the selection marquee activates making it impossible to drag and drop.

Reply:
ah, thanks for clarifying! i think that's something you'll want to take care of in your version of `cleanSpecForPreview`, by removing signals from the previews. doing this in the exporter will have downstream effects for other things, like traversing the history